src/components/MessagesList/MessagesList.vue

```diff
@@ -805,16 +805,17 @@ export default {
 					// already loading, don't do it twice
 					return
 				}
-				if (scrollTop === 0) {
-					this.displayMessagesLoader = true
-				}
+				this.displayMessagesLoader = true
 				await this.getOldMessages(false)
 				this.displayMessagesLoader = false
 
 				if (this.$refs.scroller.scrollHeight !== scrollHeight) {
-					this.$refs.scroller.scrollTo({
-						top: this.$refs.scroller.scrollTop + this.$refs.scroller.scrollHeight - scrollHeight,
-					})
+					// If was scrolled to the top, scroll for added height, otherwise browser handles it natively
+					if (scrollTop === 0) {
+						this.$refs.scroller.scrollTo({
+							top: this.$refs.scroller.scrollHeight - scrollHeight,
+						})
```

Comment:
That's the point
If before the list update scrollTop was 0, it will remain 0, so we'll miss all new messages.
If it was non-0, say, `140px`, then with updating of the list, browser will automatically correct it to `addedHeight + 140px`

Reply:
aa okay, I was focusing on another issue nvm. So the issue you are trying to fix in this PR is the scroll to bottom to include new messages in the view (currently new messages stay below and user needs to scroll manually)?

Reply:
It is for scroll to top, when 'old' messages are appended above current list